plugins/locationlayer/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="mapbox_plugin_blue">#4882C6</color>
+    <color name="mapbox_plugin_gray">#263D57</color>
```

Comment:
this resource is placed in a specific plugin folder, thus the name should be more unique to avoid id conflicts.